tests/test_jobs.py

```diff
@@ -0,0 +1,84 @@
+import re
+import unittest
+import unittest.mock
+import tutor.jobs as jobs
+from io import StringIO
+from tests.test import TestContext
+
+
+class JobsTests(unittest.TestCase):
+    @unittest.mock.patch("sys.stdout", new_callable=StringIO)
+    def test_initialise(self, mock_stdout: StringIO) -> None:
+        context = TestContext()
+        config = context.load_config()
+        runner = context.job_runner(config)
+        jobs.initialise(runner)
+
+        output = mock_stdout.getvalue()
+        service = re.search(r"Service: (\w*)", output)
+        commands = re.search(r"(-----)([\S\s]+)(-----)", output)
+        assert service is not None
+        assert commands is not None
```

Comment:
We should convert this to `assertIsNone` statements (likewise for other `assert ...` statements below).

Reply:
@regisb, I tried to use assertIsNone but this will still trigger the error from mypy as shown below.
![image](https://user-images.githubusercontent.com/10910968/144870790-40c7a7db-ae3a-4142-b6d8-3357081d9464.png)

The only way I know at the moment is to use `assert *** is not None` so that [mypy would be able to ascertain that the object is indeed not null.](https://stackoverflow.com/a/68471435/3104587)